integration-tests/lib/test-functions.sh

```diff
@@ -223,6 +224,19 @@ create_github_branch() {
     echo "Branch creation initiated."
 }
 
+create_github_repository() {
+    echo "Creating component repository ${component_repo_name} branch ${component_branch} from ${component_base_repo_name} branch ${component_base_branch}"
+    "${SUITE_DIR}/../scripts/copy-branch-to-repo-git.sh" "${component_base_repo_name}" "${component_base_branch}" "${component_repo_name}" "${component_branch}"
+}
+
+# Function to delete old repositories that match the component type prefix
```

Comment:
Won't this mean my collectors test messes with yours if we run them at the same time? Since they have the same `component_type` and it deletes all repositories with that prefix?

Reply:
No. This is looking for orphan repos where the cleanup never ran, etc. It also looks to see if the repo is older than REPO_AGE_CUTOFF_SECONDS (which is 2 days)

This is how the branch version worked.

Reply:
I have removed any deleting of repositories outside of the current repo being used.

Reply:
ack, thanks